runtime/compiler/control/JITaaSCompilationThread.cpp

```diff
@@ -242,9 +242,6 @@ bool handleServerMessage(JITaaS::J9ClientStream *client, TR_J9VM *fe)
    // If JVM has unloaded classes inform the server to abort this compilation
    if (compInfoPT->compilationShouldBeInterrupted())
       {
-      if (response != J9ServerMessageType::compilationCode)
```

Comment:
We should still send a message to the server to abort current compilation. Otherwise the server will hit a "message not expected" scenario.